boards/DCM/Src/App/states/App_InitState.c

```diff
@@ -60,6 +60,12 @@ static void InitStateRunOnTick100Hz(struct StateMachine *const state_machine)
     return;
 #endif
```

Comment:
out of scope but i dont think we need this DEBUG guard anymore ?

Reply:
Sure, I'll take it out

Reply:
done